docs/standard/ipn.rst

```diff
@@ -46,13 +46,13 @@ Using PayPal Standard IPN
                "notify_url": "https://www.example.com" + reverse('paypal-ipn'),
                "return_url": "https://www.example.com/your-return-location/",
                "cancel_return": "https://www.example.com/your-cancel-location/",
-
-           }
-
-           # Create the instance.
-           form = PayPalPaymentsForm(initial=paypal_dict)
-           context = {"form": form}
-           return render(request, "payment.html", context)
+               "custom": "Upgrade all users!",  # Custom command to correlate to some function later (optional)
+            }
```

Comment:
Indentation error here, and below, which is making the diff much noisier than it needs to be